xbmc/cores/DataCacheCore.h

```diff
@@ -70,6 +78,10 @@ class CDataCacheCore
   bool GetVideoRender();
   void SetPlayTimes(time_t start, int64_t current, int64_t min, int64_t max);
   void GetPlayTimes(time_t &start, int64_t &current, int64_t &min, int64_t &max);
+  void SetCutList(const std::vector<EDL::Cut>& cutList);
+  std::vector<EDL::Cut> GetCutList();
+  void SetChapters(const std::vector<std::pair<std::string, int64_t>>& chapters);
+  std::vector<std::pair<std::string, int64_t>> GetChapters();
```

Comment:
const?

Reply:
I copied the style of the rest of the class where all getters are not const. But yeah, why not make it better with new code.